base/business/learning_units/edition.py

```diff
@@ -597,3 +599,13 @@ def create_learning_unit_year_creation_message(learning_unit_year_created):
     return _("Learning Unit <a href='%(link)s'> %(acronym)s (%(academic_year)s) </a> "
              "successfuly created.") % {'link': link, 'acronym': learning_unit_year_created.acronym,
                                         'academic_year': learning_unit_year_created.academic_year}
+
+
+def update_partim_acronym(acronym_full, luy_to_update):
```

Comment:
```suggestion
def update_partims_acronym(acronym_full, luy_to_update):
```
car on update l'acronym de tous les partims dans cette fonction.